docker/wc-admin-php-test-suite/Dockerfile

```diff
@@ -13,33 +15,10 @@ RUN apk --no-cache add \
 	npm \
 	subversion \
 	unzip \
-	wget \
-	php7 \
-	php7-bcmath \
-	php7-ctype \
-	php7-curl \
-	php7-dom \
-	php7-exif \
-	php7-json \
-	php7-mcrypt \
-	php7-opcache \
-	php7-openssl \
-	php7-mbstring \
-	php7-mysqli \
-	php7-pcntl \
-	php7-pdo \
-	php7-pdo_pgsql \
-	php7-pdo_sqlite \
-	php7-pdo_mysql \
-	php7-phar \
-	php7-session \
-	php7-soap \
-	php7-tokenizer \
-	php7-xdebug \
-	php7-xml \
-	php7-xmlreader \
-	php7-zip \
-	php7-zlib
+	wget
+	
+COPY install-php-version.sh /tmp/  
+RUN chmod u+x /tmp/install-php-version.sh && /tmp/install-php-version.sh
```

Comment:
It seems easier to handle the php version logic in a bash script for now, although I am open to suggestions

Reply:
This sounds good. Since we're at the end of cool down I think it makes sense to try this approach out and we can iterate on it later if we need to.